### PR TITLE
Add CI for testnode that starts up nitro test node and rans l2 transa…

### PR DESCRIPTION
**Pull request by anodar**
…ctions

### CODE REVIEW

File: test-node.bash [hunk 1]
```diff
@@ -38,6 +38,7 @@ l3_token_bridge=false
 batchposters=1
 devprivkey=b6b15c8cb491557369f3c7d2c287b053eb229daa9c22138887752191c9520659
 l1chainid=1337
+runL2Txs=false
```
Comment (tsahee) [inline test-node.bash:41]: I prefer not to add options here if we can avoid it.. already messy enough

File: .github/workflows/testnode.bash [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# The script starts up the test node (with timeout 1 minute), with option to
+# run l2 transactions to make sure node is working
+
+timeout 60 ${{ github.workspace }}/nitro-testnode/test-node.bash --init --dev || exit_status=$?
```
Comment (tsahee) [inline .github/workflows/testnode.bash:5]: this doesn't use runl2Txs
Also - better if we can have all relevant logic here.
you can:
* add --detach when first callinf test-node
* wait for a little with sleep
* call e.g. test-node.sh script  send-l2 --ethamount 100 --to user_l2user --wait

**Review by tsahee — CHANGES_REQUESTED** [review]

File: test-node.bash (hunk 1 above)
Comment (anodar) [reply test-node.bash]: Done.

File: .github/workflows/testnode.bash (hunk 2 above)
Comment (anodar) [reply .github/workflows/testnode.bash]: Done.

**Comment by anodar** [comment]
Moved this to [new pull request](https://github.com/OffchainLabs/nitro-testnode/pull/44)

**Pull request closed without merging**